kolibri/content/serializers.py

```diff
@@ -77,7 +76,7 @@ class ContentNodeListSerializer(serializers.ListSerializer):
 
     def to_representation(self, data):
 
-        if 'request' not in self.context or not isinstance(self.context['request'].user, FacilityUser):
+        if self.context['request'].user.is_facility_user:
```

Comment:
Is it still possible that `request` won't be in the context? Not sure whether there was a reason that check was added. If so, may be better to just replace the part after the `or`.

Oh.. also, the logic seems to have been inverted? Now it returns `{}` when it *is* a FacilityUser.